### PR TITLE
chore: phone flow feedback

### PR DESCRIPTION
**Pull request by UncleSamtoshi**
- enable pasting phone number and phone code
- disable certain countries
- auto focus country input

### CODE REVIEW

File: app/i18n/i18n-types.ts [hunk 1]
```diff
@@ -5003,7 +5007,7 @@ export type TranslationFunctions = {
 		 */
 		upgrade: () => LocalizedString
 		/**
-		 * Log out and Clear all Local Data
+		 * Log out and lear all local data
```
Comment (nicolasburtey) [inline app/i18n/i18n-types.ts:5010]: Typo probably

**Review by nicolasburtey — COMMENTED** [review]
One comment

File: app/i18n/i18n-types.ts (hunk 1 above)
Comment (UncleSamtoshi) [reply app/i18n/i18n-types.ts:5010]: Jeez you're fast!

File: app/screens/home-screen/home-screen.tsx [hunk 2]
```diff
@@ -143,7 +143,8 @@ export const HomeScreen: React.FC = () => {
 
   const activateWallet = () => {
     setModalVisible(false)
-    navigation.navigate("phoneFlow")
+    // fixes a screen flash from closing the modal to opening the next screen
```
Comment (nicolasburtey) [inline app/screens/home-screen/home-screen.tsx:146]: did this commit not fix this issue? https://github.com/GaloyMoney/galoy-mobile/commit/1b8d42e52981f6f7317b6f89b6ef00500a22af43

**Pull request merged**